privacy-configuration/tests.json

```diff
@@ -626,5 +626,139 @@
                 "exceptPlatforms": []
             }
         ]
+    },
+    "ampFormats": {
+        "name": "A URL is able to be extracted from a google formatted URL.",
+        "desc": "Those tests use config4_reference.json where the trackingLinks feature contains regexes",
+        "referenceConfig": "config4_reference.json",
+        "tests": [
+            {
+                "name": "A URL is able to be extracted from a google formatted URL.",
+                "ampUrl": "https://www.google.com/amp/s/www.example.com/some/article",
+                "expectUrl": "https://www.example.com/some/article",
+                "exceptPlatforms": [
+                    "web-extension",
+                    "safari-extension",
+                    "android-browser",
+                    "macos-browser",
+                    "windows-browser"
```

Comment:
you don't have to put on the exception list platforms that don't implement this feature - it's mostly meant for tests that we know would fail on some platforms and we can't fix it (e.g. GPC headers on subrequests). (I know I did the same in my tests - sorry about that, I mean to fix it.)